### PR TITLE
add new command karmadactl explain

### PR DESCRIPTION
**Pull request by hulizhe**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
add new command karmadactl explain
```bash
$ karmadactl explain explain pods --operation-scope members --cluster member1
KIND:       Pod
VERSION:    v1

DESCRIPTION:
    Pod is a collection of containers that can run on a host. This resource is
    created by clients and scheduled onto hosts.
    
FIELDS:
  apiVersion    <string>
    APIVersion defines the versioned schema of this representation of an object.
    Servers should convert recognized schemas to the latest internal value, and
    may reject unrecognized values. More info:
    https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources

  kind  <string>
    Kind is a string value representing the REST resource this object
    represents. Servers may infer this from the endpoint the client submits
    requests to. Cannot be updated. In CamelCase. More info:
    https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds

  metadata      <ObjectMeta>
    Standard object's metadata. More info:
    https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
... ...

$ karmadactl explain explain pods                                            
KIND:       Pod
VERSION:    v1

DESCRIPTION:
    Pod is a collection of containers that can run on a host. This resource is
    created by clients and scheduled onto hosts.
    
FIELDS:
  apiVersion    <string>
    APIVersion defines the versioned schema of this representation of an object.
    Servers should convert recognized schemas to the latest internal value, and
    may reject unrecognized values. More info:
    https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources

  kind  <string>
    Kind is a string value representing the REST resource this object
    represents. Servers may infer this from the endpoint the client submits
    requests to. Cannot be updated. In CamelCase. More info:
    https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds

  metadata      <ObjectMeta>
    Standard object's metadata. More info:
......


$ karmadactl explain pods --operation-scope members                  
error: must specify a member cluster
```

**Which issue(s) this PR fixes**:
Parts of #5249

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
`karmadactl`: add a new command `karmadactl explain` to describe fields and structure of various resources in Karmada control plane or a member cluster. 
```

### CODE REVIEW

**Comment by zhzhuang-zju** [comment]
/lgtm
cc @RainbowMango

File: pkg/karmadactl/explain/explain.go [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+		# Get the documentation of a specific field of a resource in member cluster member1
+		%[1]s explain pods.spec.containers --operation-scope=members --cluster=member1
+		
+		# Get the documentation of resources in different format in Karmada control plane
+		%[1]s explain deployment --output=plaintext-openapiv2`)
+	plaintextTemplateName = "plaintext"
+)
+
+// NewCmdExplain new describe command.
```
Comment (RainbowMango) [inline pkg/karmadactl/explain/explain.go:61]: describe?

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**